pvlib/temperature.py

```diff
@@ -125,6 +125,18 @@ def sapm_cell(poa_global, temp_air, wind_speed, a, b, deltaT,
     | glass/polymer | insulated back | -2.81 | -0.0455 | 0                   |
     +---------------+----------------+-------+---------+---------------------+
 
+    Mounting cases can be described in terms of air flow across and around the
+    rear-facing surface of the module:
+
+    * "open rack" refers to mounting that allows relatively free air flow.
+      This case is typical of ground-mounted systems on fixed racking or
+      single axis trackers.
+    * "close roof" refers to limited or restricted air flow. This case is
+      typical of roof-mounted systems with some gap behind the module.
```

Comment:
We have a naming inconsistency: `close_mount` in `TEMPERATURE_MODEL_PARAMETERS` but `close_roof` here.  The original report calls it `close roof mount`.  I think we should change the docs to match the code.